packages/alfa-rules/src/common/predicate/is-marked-as-decorative.ts

```diff
@@ -0,0 +1,14 @@
+import {Role} from "@siteimprove/alfa-aria";
+import {Element} from "@siteimprove/alfa-dom";
+
+/**
+ * Check if an element is marked as decorative by looking at its role but without conflict resolution.
+ * If the result is "none" or "presentation", then the element is marked as decorative.
+ */
+export function isMarkedAsDecorative(element: Element): boolean {
```

Comment:
```suggestion
export function isDecorative(element: Element): boolean {
```

Reply:
I disagree. This corresponds to our definition of "marked as decorative" and does not check if the element is indeed decorative. It does not even check if the element has a semantic role of none/presentation.

Reply:
Then I'd call it `isMarkedDecorative()`. `MarkedAs` has an odd ring to it 🤔

Reply:
`isIntendedDecorative()` is also an option.